Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.cs

```diff
@@ -116,14 +117,33 @@ private void BladeOnVisibilityChanged(object sender, Visibility visibility)
                 BladeOpened?.Invoke(this, blade);
                 ActiveBlades.Add(blade);
                 UpdateLayout();
-                GetScrollViewer()?.ChangeView(_scrollViewer.ScrollableWidth, null, null);
+
+                if (BladeMode == BladeMode.Normal)
+                {
+                    GetScrollViewer()?.ChangeView(_scrollViewer.ScrollableWidth, null, null);
+                }
+
+                if (BladeMode == BladeMode.Fullscreen)
+                {
+                    for (int x = 1; x < ActiveBlades.Count; x++)
+                    {
+                        await ScrollAsync(GetScrollViewer());
+                    }
+                }
+
                 return;
             }
 
             BladeClosed?.Invoke(this, blade);
             ActiveBlades.Remove(blade);
         }
 
+        private async Task ScrollAsync(ScrollViewer scrollViewer)
+        {
+            scrollViewer.ChangeView(scrollViewer.ScrollableWidth, null, null);
+            await Task.Delay(500);
```

Comment:
Because of the state of blade item in Fullscreen mode, I absolutely need to wait a certain amount of time to be sure the Scroll is ended until we can scroll to the next Fullscreen blade item.

I tried with Delay(250) but sometimes it broke because the Scroll take too much time.